dialog/fileitem.go

```diff
@@ -24,7 +23,7 @@ type fileDialogItem struct {
 
 	icon fyne.CanvasObject
 	name string
-	path string
+	path fyne.URI
```

Comment:
Calling this path makes some of the usages look strange - is there another name that could work?

Reply:
Maybe `parent`?

Reply:
Isn't it the item URI rather than the parent?
id, location, res or others like that maybe

Reply:
I went with location in 78c6984e. I think we should start moving from `path` to `location/loc`. I could probably be sold on `reference`/`ref` if someone feels strongly.